src/modules/scraper/service/CheckMissedFillEventsCron.ts

```diff
@@ -54,7 +54,9 @@ export class CheckMissedFillEventsCron {
       .where("d.status = :status", { status: "pending" })
       .andWhere("d.outputTokenAddress is not null")
       .andWhere("d.depositDate <= NOW() - INTERVAL '5 minutes'")
+      .andWhere("d.fillDeadline >= NOW()") // ignore expired deposits
       .andWhere("j.id is null")
+      .orderBy("d.depositDate", "DESC")
```

Comment:
ah ok to latest first. & we're dropping the expired deposits

Reply:
Yes exactly, we don't have a 'expired' status in the scraper so that's why i'm using fillDeadline.
But did you have any question regarding this?